spec/lib/submission_builder/ty2024/states/nj/nj_return_xml_spec.rb

```diff
@@ -8,6 +8,50 @@
     let!(:submission_efile_device_info) { create :state_file_efile_device_info, :submission, :filled, intake: intake }
     let(:xml) { Nokogiri::XML::Document.parse(described_class.build(submission).document.to_xml) }
 
+    describe "XML schema" do
+
+      context "with one dep" do
+        let(:intake) { create(:state_file_nj_intake, municipality_code: "0101", raw_direct_file_data: StateFile::XmlReturnSampleService.new.read('nj_zeus_one_dep')) }
```

Comment:
[dust] Why specify municipality code here?

Reply:
Municipality is a required field in the XML, and by default it is nil (until the user enters it in the app) so to simulate that and get the test to pass, it has to have a value in the test data input.